go.mod

```diff
@@ -1,33 +1,122 @@
 module github.com/osmosis-labs/osmosis/v3
```

Comment:
Sorry, silly question. Being that we're on v7 of osmosis, Osmosis should technically be on v7 in the repo too, no?

Reply:
this is a PR to the v4 branch

Reply:
derppp sorry :p